dev/source/docs/ros2-sitl.rst

```diff
@@ -4,7 +4,7 @@
 ROS 2 with SITL
 ===============
 
-Once ROS2 is correctly :ref:`installed <ros2>`, source your workspace and launch Ardupilot SITL with ROS 2! 
+Once ROS2 is correctly :ref:`installed <ros2>`, and SITL is also :ref:`installed <sitl-simulator-software-in-the-loop>`, source your workspace and launch Ardupilot SITL with ROS 2! 
```

Comment:
All looks fine. From the discord discussions with users the main thing I'd want to emphasise is that users should have a working ArduPilot SITL environment (which requires all deps be installed) before attempting to build and run with ROS 2 integration.

Reply:
Should I change the wording here to be `installed and working` instead of `installed`?